src/content/docs/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,671 @@
+
+When part of a [content collection](/en/guides/content-collections/), Markdown and MDX files will live in folders within `src/content/` and you will [generate those pages dynamically](/en/guides/content-collections/#generating-routes-from-content).
+
+### Migrating Tests
+
+As Astro outputs raw HTML, it is possible to write end-to-end tests using the output of the build step. Any end-to-end tests written previously might work out-of-the-box, if you have been able to match the markup of your Nuxt site. Testing libraries such as Jest and Vue Testing Library can be imported and used in Astro to test your Vue components.
+
+See Astro's [testing guide](/en/guides/testing/) for more.
+
+## Reference: Convert NuxtJS Syntax to Astro
+
+### Nuxt Local Variables to Astro
+
+To use local variables in an Astro component's HTML, change the set of two curly braces to one set of curly braces:
+
+```astro title="src/components/Component.astro" del={3} add={4}
+---
+const message = "Hello!"
+---
+<p>{{message}}</p>
+<p>{message}</p>
+```
+
+### Nuxt Property Passing to Astro
+
+To bind an attribute or component property in an Astro component, change this syntax to the following:
+
+```astro title="src/components/Component.astro" del={3-7} ins={9-11}
+---
+---
+<p v-bind:aria-label="message">...</p>
+<!-- Or -->
+<p :aria-label="message">...</p>
+<!-- Also support component props -->
+<Header title="Page"/>
+
+<p aria-label={message}>...</p>
+<!-- Also support component props -->
+<Header title={"Page"}/>
+```
+
+### Nuxt Links to Astro
+
+Convert any Nuxt `<NuxtLink to="">` components to HTML `<a href="">` tags. 
+
+```astro del={1} ins={2}
+<NuxtLink to="/blog">Blog</Link>
+<a href="/blog">Blog</a>
+```
+
+Astro does not use any special component for links, although you are welcome to build custom link components. You can then import and use this `<Link>` just as you would any other component.
+
+```astro title="src/components/Link.astro"
+---
+const { to } = Astro.props
+---
+<a href={to}><slot /></a>
+```
+
+### Nuxt Imports to Astro
+
+If necessary, update any [file imports](/en/guides/imports/) to reference relative file paths exactly. This can be done using [import aliases](/en/guides/typescript/#import-aliases), or by writing out a relative path in full. 
+
+Note that `.astro` and several other file types must be imported with their full file extension.
+
+```astro title="src/pages/authors/Fred.astro" ".astro"
+---
+import Card from `../../components/Card.astro`;
+---
+<Card />
+```
+
+### Nuxt Dynamic Page Generation to Astro
```

Comment:
We should probably also say in this section that you can use SSR in Astro. (And, specify that `getStaticPaths()` is the non-SSR option)

Reply:
https://github.com/withastro/docs/pull/2309/files#r1121157055